wercker.yml

```diff
@@ -28,6 +30,27 @@ ci:
         code: |
           bash <(curl -s https://codecov.io/bash)
 
+# build and test UI
+ci-ui:
+  box: node
+  steps:
+    - script:
+        name: install and build
+        code: |
+          cd ui
+          npm install
+          npm build
+    - script:
+        name: run tests
+        code: |
+          cd ui
+          npm test
+    - script:
+        name: clean up
+        code: |
+          cd ui
+          rm -rf ./node_modules
```

Comment:
We don't do anything with the CI containers after they're done so we don't need to worry about removing node_modules or cleanup etc.